Modelica/Clocked/package.mo

```diff
@@ -68,7 +68,7 @@ revisions="<html>
 
 <tr><td></td><td>Several releases as <em>Modelica_Synchronous</em> library.</td></tr>
 
-<tr><td> 2019 </td><td>The <em>Modelica_Synchronous</em> library is included as <em>Modelica.Clocked</em> in the Modelica Standard Library 4.0.0.</td></tr>
+<tr><td> 2019 </td><td>The <em>Modelica_Synchronous</em> library is included as <em>Modelica.Clocked</em> in the Modelica Standard Library 4.1.0.</td></tr>
```

Comment:
```suggestion
<tr><td> 2019 </td><td>The <em>Modelica_Synchronous</em> library is included as <em>Modelica.Clocked</em> in the Modelica Standard Library.</td></tr>
```
Just drop the version here, for two reasons:
- No need to update every time.
- It's a revision stating that this happened in 2019, clearly it wasn't added to 4.1.0 in 2019.

An alternative would be to replace it with "starting from 4.0.0" - but why bother?